code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm

```diff
@@ -471,10 +471,10 @@
 		/datum/reagent/liquidgibs = -2)
 
 	virus_suspectibility = 0
-	resulting_atoms = list(/mob/living/simple_animal/hostile/netherworld = 1)
+	resulting_atoms = list(/mob/living/basic = 1)
```

Comment:
This is an abstract typepath, don't use it in this context... maybe just add all four mobs as a weighted list? Unsure what this code does but I don't think anything good will happen if you do it this way